.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,48 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-2019, ubuntu-latest]
+        dotnetcore:
+          - 2.2.207
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ matrix.dotnetcore }}
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Test
+        run: dotnet test --no-restore --verbosity normal
+
+  release:
+    runs-on: ubuntu-latest
+    # if: github.ref == 'refs/heads/master'
```

Comment:
this will need to be un-commented. Not sure how the release strategy should work